test/fixtures.py

```diff
@@ -317,21 +336,55 @@ def open(self):
             tries += 1
         else:
             raise Exception('Failed to start KafkaInstance before max_timeout')
-        self.out("Done!")
         self.running = True
+        self.out("Kafka started!")
+
+    def open(self):
+        if self.running:
+            self.out("Instance already running")
+            return
+
+        # Create directories
+        if self.tmp_dir is None:
+            self.tmp_dir = py.path.local.mkdtemp()
+        self.tmp_dir.ensure(dir=True)
+        self.tmp_dir.ensure('logs', dir=True)
+        self.tmp_dir.ensure('data', dir=True)
+
+        self.out("Running local instance...")
+        log.info("  host       = %s", self.host)
+        log.info("  port       = %s", self.port or '(auto)')
+        log.info("  transport  = %s", self.transport)
+        log.info("  broker_id  = %s", self.broker_id)
+        log.info("  zk_host    = %s", self.zk_host)
+        log.info("  zk_port    = %s", self.zk_port)
+        log.info("  zk_chroot  = %s", self.zk_chroot)
+        log.info("  replicas   = %s", self.replicas)
+        log.info("  partitions = %s", self.partitions)
+        log.info("  tmp_dir    = %s", self.tmp_dir.strpath)
```

Comment:
I wonder whether it's better to have these as separate log entries or as a single log entry that just has newlines? I'm not actually sure.

Reply:
I personally prefer the way it is. It makes the code more readable, IMO.